zerver/lib/actions.py

```diff
@@ -4715,10 +4717,25 @@ def do_delete_messages(realm: Realm, messages: Iterable[Message]) -> None:
         # We exclude long-term idle users, since they by definition have no active clients.
         subscribers = subscribers.exclude(user_profile__long_term_idle=True)
         subscribers_ids = [user.user_profile_id for user in subscribers]
+        users_ids = subscribers_ids
         users_to_notify = list(map(subscriber_info, subscribers_ids))
 
+    active_push_messages = list(UserMessage.objects.filter(
+        message_id__in=message_ids,
+        user_profile_id__in=users_ids,
+    ).extra(
+        where=[UserMessage.where_active_push_notification()],
+    ).values_list('message_id', 'user_profile_id'))
```

Comment:
```suggestion
    ).values('message_id', 'user_profile_id'))
```

will make things more readable by letting us use keys instead of `0, 1` indices below